testing/cachefilesd-inotify/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Contributor: Jake Buchholz <tomalok@gmail.com>
+# Maintainer: Jake Buchholz <tomalok@gmail.com>
+pkgname=cachefilesd-inotify
+pkgver=0.11.0
+pkgrel=0
+pkgdesc="Userspace daemon acting as a backend for FS-Cache (uses INOTIFY instead of DNOTIFY)"
+url="https://gitlab.com/tomalok/cachefilesd-inotify"
+arch="all"
+license="GPL"
```

Comment:
That isn't a valid license identifier as of https://spdx.org/licenses

Reply:
Updated to `"GPL-2.0-or-later"` as the RPM spec file in the source says "GPLv2+"